assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/NoPricesBanner.tsx

```diff
@@ -4,11 +4,19 @@ import { __ } from '@wordpress/i18n';
 import { Banner } from '@application/ui/display';
 
 import AddDefaultPricesButton from '../buttons/AddDefaultPricesButton';
+import { TicketPriceCalculatorProps } from './TicketPriceCalculator';
 
-const NoPricesBanner = () => {
-	const description = __(
-		'Click the button below to load your default prices into the calculator. Additional ticket price modifiers can be added or removed. Click the save button below to assign which dates this ticket will be available for purchase on.'
+interface Props extends Pick<TicketPriceCalculatorProps, 'context'> {}
+
+const NoPricesBanner: React.FC<Props> = ({ context }) => {
+	let description = __(
+		'Click the button below to load your default prices into the calculator. Additional ticket price modifiers can be added or removed.'
 	);
+
+	if (context === 'futureUseCase') {
+		description += __(' Click the save button below to assign which dates this ticket will be available for purchase on.');
```

Comment:
```suggestion
		description += ' ' + __('Click the save button below to assign which dates this ticket will be available for purchase on.');
```

As I said, translators can miss that space at the ends. So it's better to take it out.